common_benchmark_suite/openxla/benchmark/models/tflite/tflite_model.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import os
+import urllib
+
+
+class TfliteModel:
```

Comment:
Not very sure about the naming convention, maybe TFLiteModel?

Reply:
Changed